diffsync/logging.py

```diff
@@ -35,18 +38,45 @@ def enable_console_logging(verbosity=0):
     else:
         logging.basicConfig(format="%(message)s", level=logging.DEBUG)
 
+    processors = [
+        structlog.stdlib.filter_by_level,  # <-- added
+        structlog.stdlib.add_logger_name,
+        structlog.stdlib.add_log_level,
+        structlog.processors.TimeStamper(fmt="%Y-%m-%d %H:%M.%S"),
+        structlog.processors.StackInfoRenderer(),
+    ]
+
+    if _structlog_exception_formatter_required():
+        processors.append(structlog.processors.format_exc_info)
+
+    # ConsoleRenderer must be added after format_exc_info
+    processors.append(structlog.dev.ConsoleRenderer())
+
     structlog.configure(
-        processors=[
-            structlog.stdlib.filter_by_level,  # <-- added
-            structlog.stdlib.add_logger_name,
-            structlog.stdlib.add_log_level,
-            structlog.processors.TimeStamper(fmt="%Y-%m-%d %H:%M.%S"),
-            structlog.processors.StackInfoRenderer(),
-            structlog.processors.format_exc_info,
-            structlog.dev.ConsoleRenderer(),
-        ],
```

Comment:
Don't we need a `processors=processors,` line added here?

Reply:
Indeed, it got lost somewhere in the experimentation phase.  Added this back in.